conn2res/plotting.py

```diff
@@ -574,78 +600,40 @@ def plot_performance(
 
     # plot
     sns.lineplot(
-        data=df, x=x, y=y, hue=hue, palette=palette, markers=True,
-        legend=True, ax=ax, **kwargs)
-
-    # set axis labels
-    ax.set_xlabel('alpha', fontsize=11)
-    y_label = ' '.join(y.split('_'))
-    ax.set_ylabel(y_label, fontsize=11)
+        data=df, x=x, y=y, hue=hue, palette=palette, dashes=False,
+        legend=False, markers=True, ax=ax, **kwargs)
 
     # set legend
-    plt.legend(loc='upper right', fontsize=8)
+    try:
+        lg_defaults = {'labels': kwargs['hue_order']}
+    except:
+        lg_defaults = {'labels': list(pd.unique(df[hue]))}
+    lg_defaults.update(**lg_params)
+    ax.legend(handles=ax.lines, **lg_defaults)
+
+    # set axis properties
+    axes_defaults = {'xlabel': x, 'ylabel': ' '.join(y.split('_'))}
+    if title is not None:
+        axes_defaults.update(title=title)
+    axes_defaults.update(**ax_params)
+    ax.set(**axes_defaults)
 
     # set title
-    if title is not None:
-        plt.title(title, fontsize=12)
+    # if title is not None:
+    #     fig.suptitle(title)
```

Comment:
Again, I would suggest to uncomment here and delete lines 616-617 to set the title at the level of the Figure

Reply:
I'll do so!